CASC/PortfolioMode.cpp

```diff
@@ -155,6 +165,10 @@ bool PortfolioMode::performStrategy(Shell::Property* property)
   Schedule main_extra;
   Schedule fallback_extra;
 
+  // TODO this doesn't make a lot of sense to me either
+  // first make *_extra (x3 multiplier with new options) schedules
+  // then later repeat them (x2 multiplier, no new options) until timeout
+  // if this is really what we want, fine, but why?!
```

Comment:
Like the comment says. Any ideas what the logic is/was?

Reply:
@selig maybe?